### PR TITLE
fix: allow HTML multiline tag syntax

### LINKED ISSUE #46978: Problem step 6 new responsive web design? CAMPER CAFE
### Describe the Issue

when putting the opening en closing tag p on differnt lines from Est. 2020. I get a failure response although this seems to be acceptable in html.

Only when I put both p tags on the same line as Est. 2020, I get the answer right.

### Affected Page

https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-basic-css-by-building-a-cafe-menu/step-6

### Your code
```
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8" />
    <title>Cafe Menu</title>
  </head>
  <body>
<h1>CAMPER CAFE</h1>
    **<p>
      Est. 2020
    </p>**
 </body>
</html>
```

### Expected behavior

This should pass as a correct answer as this is acceptable in html.

### Screenshots

step6

### System

- Device: [ Laptop]
- OS: [ Windows 10]
- Browser: [Vivaldi]
- Version: [5.3.2679.68]

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by gabrielrochamd**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #46978

### CODE REVIEW

**Review by Sboonny — APPROVED** [review]
#### :sparkles: :tada: **AWESOME!** :tada: :sparkles:

 Hi @gabrielrochamd, thanks for this pull request and for contributing to the code-base for the first time. We are looking forward to more contributions from you in the future.

 Cheers & happy contributing!

File: curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f332a88dc25a0fd25c7687a.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ assert($('p')[0].previousElementSibling.tagName === 'H1');
 Your `p` element should have the text `Est. 2020`.
 
 ```js
-assert(code.match(/<p>Est. 2020<\/p>/i));
+assert($('p')[0].innerText === 'Est. 2020');
```
Comment (Sboonny) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f332a88dc25a0fd25c7687a.md:41]:
```suggestion
assert(document.querySelector("p").innerText === "Est. 2020")
```

**Review by Sboonny — CHANGES_REQUESTED** [review]
Sorry @gabrielrochamd, should have noticed it, but we are changing from jQuery to JavaScript

**Comment by gabrielrochamd** [comment]
@Sboonny I accidentally accepted the changes without editing commit message. Should I amend it?

**Comment by Sboonny** [comment]
@gabrielrochamd all good thank you 👍

**Review by Sboonny — APPROVED** [review]
LGTM 🎉, thank you for your contribution to the page! 👍.

**Review by naomi-lgbt — APPROVED** [review]
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Comment by naomi-lgbt** [comment]
Congrats on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We are happy to accept these changes and look forward to future contributions. 📝

**Pull request merged**